bin/pie-my-vulns.js

```diff
@@ -13,10 +14,12 @@ const EXIT_CODE_VULNS_NONE = 0
 const reportsList = [SeverityReporter, DependencyTypeReporter, RemediationTypeReporter]
 
 async function main() {
+  const argv = parseArgs(process.argv.slice(2))
+  const { directory } = argv
   let vulnerabilitiesResult
   const audit = new Audit()
   try {
-    vulnerabilitiesResult = await audit.test()
+    vulnerabilitiesResult = await audit.test(directory)
```

Comment:
I assume we'll be adding more and more options here so how about we pass in an object for `audit.test()`, such as:
```
audit.test({
 directory: directory
})
```

or in an object's short-form if you prefer.